plugins/arDominionB5Plugin/js/imageflow.js

```diff
@@ -1,34 +1,33 @@
-(function ($) {
+(() => {
```

Comment:
This should be `($ => {`. Isn't it complaining about line 3 without passing `$`?